cmd/virsnap/clean.go

```diff
@@ -2,46 +2,49 @@
 // Licensed under the MIT License. You have obtained a copy of the License at
 // the "LICENSE" file in this repository.
 
-// Package cmd implements the handlers for the different command line arguments.
```

Comment:
I think the comment might be quite helpful for people that do not know the default golang project structure well. Re-Add the comment, please.

Reply:
Fixed